ML Beetle info pages (#4)
Imageomics/HDR-SMood-Challenge: pages/overview.md

# Beetles as Sentinel Taxa: Predicting drought conditions from NEON specimen imagery

> ### Brought to you by Imageomics Institute as part of the 2025 HDR ML Challenge

Reviewer: ```suggestion
### Brought to you by Imageomics Institute and NEON as part of the 2025 HDR ML Challenge
```

The [National Ecological Observatory Network (NEON)](https://www.neonscience.org/) provides unprecedented, standardized ecological data across the United States, including systematic collections of carabid beetles ([DP1.10022.001](https://data.neonscience.org/data-products/DP1.10022.001)) from diverse terrestrial ecosystems ([Thorpe et al., 2016]( https://doi.org/10.1002/ecs2.1627)). These collections, housed in the [NEON Biorepository](https://biorepo.neonscience.org), include high-resolution images of individual specimens. This challenge leverages this unique resource to explore a novel approach to environmental monitoring.

## Setup Overview

> The goal of this challenge is to predict the drought status at a location on a specific date based on images of carabid beetle specimens collected at the location on that date. This will be an out-of-sample challenge because training data will be assembled from one site of locations, and test data will assembled from a different set of locations. All locations will be NEON sites.

Reviewer 2: ```suggestion
The goal of this challenge is to predict the drought status at a location on a specific date based on images of carabid beetle specimens collected at the location on that date. This will be an out-of-sample challenge because training data will be assembled from one site of locations, and test data will be assembled from a different set of locations. All locations will be NEON sites.
```

> The goal of this challenge is to predict the drought status at a location on a specific date based on images of carabid beetle specimens collected at the location on that date. This will be an out-of-sample challenge because training data will be assembled from one site of locations, and test data will be assembled from a different set of locations. All locations will be NEON sites.

Reviewer: ```suggestion
The goal of this challenge is to predict the drought status at a location on a specific date based on images of carabid beetle specimens collected at the location on that date. This will be an out-of-sample challenge because training data will be assembled from one set of locations, with test data coming from a different set of locations. All locations will be NEON sites.
```

> This challenge will provide insight about how well carabid beetles can serve as indicator taxa to understand local drought conditions. This challenge will help ecologists better understand how inter- and intra-specific variation in carabid traits correspond with local environmental conditions, and over what time scales. We expect trait variations among specimens can be detected in the imagery and linked to drought metrics using AI/ML models. If these models perform better than benchmark models (e.g., historical mean drought condition at a site for the given day-of-year, or models using only metadata), it suggests there is ecologically relevant information that can be extracted from specimen images and linked to environmental data.

Reviewer 2: ```suggestion
This challenge will provide insight about how well carabid beetles can serve as indicator taxa to understand local drought conditions. It will also help ecologists better understand how inter- and intra-specific variation in carabid traits correspond with local environmental conditions, and over what time scales. We expect trait variations among specimens can be detected in the imagery and linked to drought metrics using AI/ML models. If these models perform better than benchmark models (e.g., historical mean drought condition at a site for the given day-of-year, or models using only metadata), it suggests there is ecologically relevant information that can be extracted from specimen images and linked to environmental data.
```

### Algorithm requirement

> Participants are tasked with predicting the Standardized Precipitation Evapotranspiration Index (SPEI) metric, which represents cumulative drought conditions over a time specified time window. For this challenge, participants will be provided with images of beetle specimens collected during a sampling event and they must predict values of the SPEI metric for three time windows:

Reviewer: ```suggestion
Participants are tasked with predicting the Standardized Precipitation Evapotranspiration Index (SPEI) metric, which represents cumulative drought conditions over a specified time window. For this challenge, participants will be provided with images of beetle specimens collected at a particular location during a sampling event. Based on these images, they must predict values of the SPEI metric for three time windows:
```

> - spei_30d is drought conditions calcualted from data for the 30 day window preceding sample collection
> - spei_1y is the drought condition calculated from the year preceding sample collection
> - spei_2y is the drought condition calculated from the two years preceding sample collection

Reviewer: ```suggestion
- `spei_30d`: drought condition calculated from data for the 30 day window preceding sample collection.
- `spei_1y`: drought condition calculated from the year preceding sample collection.
- `spei_2y`: drought condition calculated from the two years preceding sample collection.
```

### Training data
The training data comprises images from beetle specimens and a metadata file that indicates the location and date each specimen was collected, its scientific name, and the values of the three SPEI metrics at the sampling location on the collection date. 

### Test data

> Test data will include images of beetle specimens and limited metadata. The metadata file will indicate an anonymized sampling `event_id` associated with each specimen, along with the specimen's scientific name and the eco-climatic domain of the collection site (but not the site ID or location). The collection date and the SPEI values will not be provided in the test dataset.  

Reviewer: ```suggestion
Test data will include images of beetle specimens and limited metadata. The metadata file will indicate an anonymized sampling `eventId` associated with each specimen, along with the specimen's scientific name and the eco-climatic domain of the collection site (but not the site ID or location). The collection date and the SPEI values will not be provided in the test dataset.
```

> Test data will include images of beetle specimens and limited metadata. The metadata file will indicate an anonymized sampling `event_id` associated with each specimen, along with the specimen's scientific name and the eco-climatic domain of the collection site (but not the site ID or location). The collection date and the SPEI values will not be provided in the test dataset.  

Reviewer 3: This looks good to me, but yes, we will provide `domainID` during testing, but anonymized. In training they will have `siteID` anonymized, and `collectDate`.

> Test data will include images of beetle specimens and limited metadata. The metadata file will indicate an anonymized sampling `event_id` associated with each specimen, along with the specimens scientific name and the eco-climatic domain that of the collection site (but not the site ID or location). The collectiont date and the SPEI values will not be provided in the test dataset.  

Reviewer 2: ```suggestion
Test data will include images of beetle specimens and limited metadata. The metadata file will indicate an anonymized sampling `event_id` associated with each specimen, along with the specimen's scientific name and the eco-climatic domain of the collection site (but not the site ID or location). The collection date and the SPEI values will not be provided in the test dataset.  
```

## Timeline

> This ML Challenge starts on <<Aug 2025>>, and will run through <<January 2026??>>. Be sure to resubmit your preferred model from the development phase by <<date??>> at 11:59pm [AOE](https://www.timeanddate.com/time/zones/aoe); it will then be run on the final test sets. Only one submission will be run against the test sets to determine your final score in the challenge.

Reviewer: ```suggestion
This ML Challenge starts on August 13, 2025>>, and will run through October 31, 2025. Be sure to resubmit your preferred model from the development phase by October 31, 2025 at 11:59pm [AOE](https://www.timeanddate.com/time/zones/aoe); it will then be run on the final test sets. Only one submission will be run against the test sets to determine your final score in the challenge.
```
We _may_ extend the deadline by a week, but as of now, this is the timeline for the challenge.

> This ML Challenge starts on <<Aug 2025>>, and will run through <<January 2026??>>. Be sure to resubmit your preferred model from the development phase by <<date??>> at 11:59pm [AOE](https://www.timeanddate.com/time/zones/aoe); it will then be run on the final test sets. Only one submission will be run against the test sets to determine your final score in the challenge.

Reviewer 2: Dates need to be filled in for the placeholders.

> This challenge was generated based on the [CodaBench Iris Sample Bundle](https://github.com/codalab/competition-examples/tree/master/codabench/iris/bundle); full formatting and challenge design process can be found in the [challenge repo on GitHub](https://github.com/Imageomics/HDR-SMood-Challenge).

Reviewer: ```suggestion
This challenge was generated based on the [Imageomics/HDR-anomaly-challenge](https://github.com/Imageomics/HDR-anomaly-challenge), which used the [CodaBench Iris Sample Bundle](https://github.com/codalab/competition-examples/tree/master/codabench/iris/bundle); full formatting and challenge design process can be found in the [challenge repo on GitHub](https://github.com/Imageomics/HDR-SMood-Challenge).
```
